pages/users/change-password.tsx

```diff
@@ -0,0 +1,55 @@
+import Layout from 'components/Layout';
```

Comment:
you'll need to run prettier here
you can do so with the shortcut `cmd+option+f`